packages/melody-streams/src/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2018 trivago N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { createComponent } from './component';
+export { createState } from './operators/createState';
```

Comment:
What do you think about separating the functions in `./operators` into `helpers` and `operators`?

Actual operators are:
```
combine: Observable
mergeIntoObjects: Observable
mergeObject: Observable
```

Helpers have a different API:
```
withElement: [Element => Subscription, Subject]
withEvent: [Element => Subscription, Subject]; 

createState: [Subject, state => void, () => state];
```
And maybe use `withEvent` instead of `attachEvent` for consistency?

Reply:
I also like the name `withState`